src/utils/hooks/consoleUserSettings/useQueuedUserSettingWrite/queuedUserSettingWrite.ts

```
@@ -0,0 +1,59 @@
+
+    isRunning = true;
+
+    try {
+      while (pending) {
+        const batch = pending;
+        // Empty the queue before waiting so new writes start a fresh batch instead of joining this one.
+        pending = null;
+
+        try {
+          await batch.write(batch.value);
+          batch.resolvers.forEach(({ resolve }) => resolve());
```

Biome (2.5.0)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)